src/frontend/screens/Observation/ObservationView.js

```diff
@@ -1,50 +1,112 @@
 // @flow
 import React from "react";
-import { Text, Image, View, ScrollView, StyleSheet, Share } from "react-native";
+import { Text, View, ScrollView, StyleSheet, Share } from "react-native";
+import MapboxGL from "@react-native-mapbox-gl/maps";
 import ShareMedia from "react-native-share";
 import MaterialIcons from "react-native-vector-icons/MaterialIcons";
 
 import api from "../../api";
+import MapStyleProvider from "../../sharedComponents/MapStyleProvider";
 import FormattedCoords from "../../sharedComponents/FormattedCoords";
 import ThumbnailScrollView from "../../sharedComponents/ThumbnailScrollView";
-import { DetailsIcon, CategoryIcon } from "../../sharedComponents/icons";
+import { CategoryCircleIcon } from "../../sharedComponents/icons";
+import mapIcon from "../../images/mapeo-icon-transparent.png";
 import { formatDate, formatCoords } from "../../lib/utils";
 import { filterPhotosFromAttachments } from "../../context/DraftObservationContext";
-import { RED, WHITE } from "../../lib/styles";
+import {
+  BLACK,
+  RED,
+  WHITE,
+  DARK_GREY,
+  LIGHT_GREY,
+  MEDIUM_GREY
+} from "../../lib/styles";
 import { TouchableOpacity } from "../../sharedComponents/Touchables";
 import type { PresetWithFields } from "../../context/PresetsContext";
 import type { Observation } from "../../context/ObservationsContext";
 
-// const InsetMapView = ({ style }) => <View style={[style, styles.insetMap]} />;
-
 type ButtonProps = {
   onPress: () => any,
   color: string,
-  iconName: "delete-forever" | "share",
+  iconName: "delete" | "share",
   title: string
 };
 
+type MapProps = {
+  lon: number,
+  lat: number
+};
+
+const MapFeatures = ({ lat, lon }: MapProps) => {
+  const featureCollection = {
+    type: "FeatureCollection",
+    features: [
+      {
+        type: "Feature",
+        geometry: {
+          type: "Point",
+          coordinates: [lon, lat]
+        },
+        properties: {
+          id: Date.now()
+        }
+      }
+    ]
+  };
+  return (
+    <MapboxGL.ShapeSource id="observation-source" shape={featureCollection}>
+      <MapboxGL.SymbolLayer
+        id="observation-symbol"
+        style={{
+          iconImage: mapIcon,
+          iconSize: 0.5,
+          iconOffset: [0, -15]
+        }}
+      />
+    </MapboxGL.ShapeSource>
+  );
+};
+
+const InsetMapView = ({ lon, lat }: MapProps) => (
+  <MapStyleProvider styleURL={MapboxGL.StyleURL.Satellite}>
```

Comment:
Was following the mockup here... not sure who to defer to... can you dbl-check w Aldo? If we agree that it makes more sense to use the main map style consistently, I'll remove some of the StyleURL changes to the `MapStyleProvider` module... @gmaclennan Going forward, it might be helpful for you to leave feedback on the designs in Notion if there's anything you want to change -- will save on dev time.